### ISSUE TITLE
Svelte Guide

### ISSUE THREAD

**Issue by ayuhito** [DecliningLotus/fontsource#443]
The project is compatible with Svelte and is even used in official SvelteKit create-svelte examples, but it would be best for us to to write some formal documentation for it on the Fontsource website to make usage more well known and compatible.

I've never used Svelte and thus I'm not confident writing anything about it. It would be great if someone would help write some documentation for it and I'll do my best to assist if needed.

To add new docs, you just need to create a new Markdown file in [`website/docs/guides`](https://github.com/fontsource/fontsource/tree/main/website/docs/guides).

**Comment by vhscom** [comment]
I was able to get Fontsource fonts loading via the provided Sass mixins with SvelteKit by customizing `fontDir` (or `woff2Path` in the case of Material Icons) like:

```
@use '@fontsource/jost/scss/mixins' as Jost;
@include Jost.fontFace($fontDir: '/node_modules/@fontsource/jost/files');
```

Here are the SvelteKit instructions, which do not mention this as a setup option:
https://fontsource.org/docs/guides/sveltejs

It would be ideal to update with instructions for using the Sass mixins with SvelteKit, and ensure it works with Playwright as Playwright is a `create-svelte` setup option and, therefore, will be used by many projects.

**Comment by vhscom** [comment]
I was able to get Fontsource fonts loading via the provided Sass mixins with SvelteKit by customizing `fontDir` (or `woff2Path` in the case of Material Icons) like:

```scss
@use '@fontsource/jost/scss/mixins' as Jost;
@include Jost.fontFace($fontDir: '/node_modules/@fontsource/jost/files');
```

**EDIT:** Above works with `dev` but not with `build && preview`, unfortunately.

Here are the SvelteKit instructions, which do not mention this as a setup option:
https://fontsource.org/docs/guides/sveltejs

It would be ideal to update with instructions for using the Sass mixins with SvelteKit, and ensure it works with Playwright as Playwright is a `create-svelte` setup option and, therefore, will be used by many projects.

**Comment by ayuhito** [comment]
Closed in #575 a while ago. 👀
